companies-manifest.js

```diff
@@ -853,6 +854,27 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://swissdevjobs.ch',
+    similarWebGlobalRank: 583400,
+    name: 'SwissDev Jobs',
+    logo: SwissDevJobsLogo,
+    style: {
```

Comment:
This is weird but can you try committing with an empty object in the `style` key?

Reply:
Done, but still seems to be failing.